core/src/main/java/org/jdbi/v3/core/statement/ParsedSql.java

```diff
@@ -72,6 +72,26 @@ public String toString() {
             + '}';
     }
 
+    /**
+     * A static factory of {@link ParsedSql} instances. The first
+     * argument, i.e. SQL, must contain only positional parameters
+     * (i.e. the {@value #POSITIONAL_PARAM} character). If your SQL
+     * code contains named parameters (i.e. variables preceded by a
```

Comment:
this should actually be `e.g.`, not `i.e.`, because the colon is merely a jdbi default, not a hard fact in this context

Reply:
e.g. stands for "exempli gratia," meaning "for example"

i.e. stands for "id est," meaning "in other words"

:)

Reply:
is that to correct me or informational for sarxos? I think it should be e.g. because being preceded by a colon is just one possible syntax, no?

Maybe the first i.e. in the paragraph can be replaced by "the statement" too, to go easier on the mind...

Reply:
These "i.e." and "e.g." was removed from javadoc.